.github/workflows/build-images.yaml

```diff
@@ -123,6 +125,70 @@ jobs:
         env:
           LAST_GREEN_COMMIT: ${{ needs.determine-last-green-commit.outputs.lastGreenCommit }}
 
+  # TODO(rustielin): build these images continuously rather than on PR
+  rust-images-testing:
```

Comment:
Will this be a land blocker?

Reply:
> Will this be a land blocker?

These are not land-blocking. It will run on the same triggers for now (we can move to continuous later) though.

> How would we specify the image tag in the tests? Do we have that support already?

You would have to know the image tag beforehand. If you have the GIT_SHA, you can do like `<profile>_<features>_GIT_SHA`

Reply:
how would I use this diff to create a continuous test that uses failpoints?

Reply:
You would need to specify an `IMAGE_TAG` yourself of the format I wrote above (you can either look at the github actions runs or the list of builds in our image registries), or we can bake into the forge invocation script a flag to select these images automatically. 

The trick is that the image tags all follow a standard format, so if we know what git revision to test, we can programmatically select the image build that has failpoints

Reply:
I am not asking for individual runs,  I'm asking for continuous test.  So every run would be on a different IMAGE_TAG?

So can we bake this in forge invocation so that I can configure a test always to use inage with failpoints?

Reply:
Oh I see what you mean -- yea if the image is different for each run, you'd need a way to toggle failpoints. Let me push up something for that

Reply:
Ok just updated the wrapper script to handle `FORGE_ENABLE_FAILPOINTS`